src/pages/basic_data/app_admin/CentralImportResultItem.tsx

```diff
@@ -0,0 +1,44 @@
+import { Divider, Grid, Link, ListItem, ListItemText, Typography } from '@mui/material';
+
+interface ResultItemProps {
+  publication: any;
```

Comment:
Mulig du kan komme unna med en Partial type
```suggestion
  publication: Partial<Registration>;
```

Reply:
men kan godt ta det i neste runde